src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -19,6 +19,32 @@ function rrule(::typeof(dot), x, y)
     return dot(x, y), dot_pullback
 end
 
+#####
+##### `cross`
+#####
+
+function frule((_, Δa, Δb), ::typeof(cross), a::AbstractVector, b::AbstractVector)
+    return cross(a, b), cross(Δa, b) .+ cross(a, Δb)
+end
+# chunked mode
+function frule((_, Δa, Δb)::Tuple{<:Any,<:AbstractMatrix,<:AbstractMatrix},
```

Comment:
There aren't  (I think) any examples of dispatching on `frule` sensitivities yet. I don't think this is the right way to do it (doesn't support `Composite`).

Reply:
Yeah, I don't think that we've formalised the API for this properly yet. [This issue](https://github.com/JuliaDiff/ChainRulesCore.jl/issues/92) is still being addressed. It's gone a bit stale now, which I suspect is because the FD2 crew are okay with where it is currently.

Reply:
I am not working on FD2 at the moment, but I do plan to rewrite it sometime.